lasagne/tests/test_objectives.py

```diff
@@ -153,6 +153,35 @@ def test_binary_hinge_loss(colvect):
 
 
 @pytest.mark.parametrize('colvect', (False, True))
+def test_huber_loss(colvect):
+    from lasagne.objectives import huber_loss
+    delta = [0.5, 1.0]
```

Comment:
You can add `delta` to the test parameters and add another `parametrize`. Looks a little cleaner.